src/applications/edu-benefits/tests/components/SchoolSelectField.unit.spec.jsx

```diff
@@ -290,7 +290,7 @@ describe('<SchoolSelectField>', () => {
       name: 'testName',
       state: 'testState',
       zip: '12345',
-      country: 'United States'
+      country: 'USA'
```

Comment:
Changing this mock data to reflect what we'd actually get back from the BE that powers school search and confirm that the country will be converted to 'United States'.